scripts/gen_help_html.lua

```diff
@@ -61,6 +61,7 @@ local new_layout = {
 -- TODO: These known invalid |links| require an update to the relevant docs.
 local exclude_invalid = {
   ["'previewpopup'"] = "quickref.txt",
+  ["[count]fclose[!]"] = "windows.txt",
```

Comment:
Remove this. This string doesn't even exist in the docs.

Reply:
if remove this, `help_spec.lua` will through an error.

Reply:
I don't think that's possible, and I don't get an error.

Reply:
you are right . now it works fine. before always tell me invalid tags